yt_dlp/extractor/adobepass.py

```diff
@@ -1526,6 +1528,27 @@ def extract_redirect_url(html, url=None, fatal=False):
                         if 'Please try again.' in saml_login_page:
                             raise ExtractorError(
                                 'We\'re sorry, but either the User ID or Password entered is not correct.')
+                    else:
+                        # elif 'Please wait ...' in provider_redirect_page and "'N'== \"Y\"" in provider_redirect_page:
+                        # ABC from non-Verizon IP
+                        saml_redirect_url = self._html_search_regex(
+                            r'var\surl\s*=\s*(["\'])(?P<url>.+?)\1',
+                            provider_redirect_page,
+                            'SAML Redirect URL', group='url')
+                        saml_redirect_url = saml_redirect_url.replace(r'\/', '/')
+                        saml_redirect_url = saml_redirect_url.replace(r'\-', '-')
+                        saml_redirect_url = saml_redirect_url.replace(r'\x26', '&')
```

Comment:
r u sure there is no better way to do this?

Reply:
Not sure, I don't really want to eval it. Parsing as JSON could work if we assume double quotes, but `\x` would have to be changed to `\u00`.